docs/user/ppl/functions/json.rst

```diff
@@ -60,3 +60,107 @@ Example::
     | json scalar string  | "abc"                           | "abc"                   |
     | json empty string   |                                 | null                    |
     +---------------------+---------------------------------+-------------------------+
+
+JSON_OBJECT
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_object(key1, value1, key2, value2...)` create a json object string with key value pairs. The key must be string.
+
+Argument type: key1: STRING, value1: ANY, key2: STRING, value2: ANY ...
+
+Return type: STRING
+
+Example::
+
+    > source=json_test | eval test_json = json_object('key', 123.45) | head 1 | fields test_json
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | test_json               |
+    |-------------------------|
+    | {"key":123.45}          |
+    +-------------------------+
+
+JSON_ARRAY
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_array(element1, element2, ...)` create a json array string with elements.
+
+Argument type: element1: ANY, element2: ANY ...
+
+Return type: STRING
+
+Example::
+
+    > source=json_test | eval test_json_array = json_array('key', 123.45) | head 1 | fields test_json_array
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | test_json_array         |
+    |-------------------------|
+    | ["key",123.45]          |
+    +-------------------------+
+
+JSON_ARRAY_LENGTH
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_array_length(value)` parse the string to json array and return size, if can't be parsed, return null
+
+Argument type: value: STRING
+
+Return type: INTEGER
+
+Example::
+
+    > source=json_test | eval array_length = json_array_length("[1,2,3]") | head 1 | fields array_length
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | array_length            |
+    |-------------------------|
+    | 3                       |
+    +-------------------------+
+
+    > source=json_test | eval array_length = json_array_length("{\"1\": 2}") | head 1 | fields array_length
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | array_length            |
+    |-------------------------|
+    | null                    |
+    +-------------------------+
+
+JSON_EXTRACT
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_extract(json_string, path1, path2, ...)` it first transfer json_string to json, then extract value using paths. If only one path, return the value, otherwise, return the list of values. If one path cannot find value, return null as the result for this path. The path use "{<index>}" to represent index for array, "{}" means "{*}".
```

Comment:
> it first transfer json_string to json, then extract value using paths. If only one path, return the value, otherwise, return the list of values.

Extracts values using the specified JSON paths. If only one path is provided, it returns a single value. If multiple paths are provided, it returns a `JSON Array` in the order of the paths.

Reply:
Already update it.